pkg/zkproofs/equality_proof_transcript.go

```diff
@@ -0,0 +1,39 @@
+package zkproofs
```

Comment:
Nit: Should we rename this file to proof_transcript.go

Reply:
Done